ca/config.go

```diff
@@ -135,6 +135,13 @@ func (s *SecurityConfig) UpdateRootCA(cert, key []byte, certExpiry time.Duration
 	return err
 }
 
+// loadNewRootCA replaces the root CA with a new root CA
+func (s *SecurityConfig) loadNewRootCA(rootCA *RootCA) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.rootCA = rootCA
+}
```

Comment:
Yes, sorry, my comment was saying there wasn't any.  I originally thought it'd make sense to have a single function save all the needed files, but it's not like the writes atomic so it shouldn't really matter.  Then I started wondering whether it *should* be atomic.  :)